Nodejs/Product/Nodejs/Snippets/1033/Nodejs/cberr.snippet

```diff
@@ -0,0 +1,30 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<CodeSnippets
+    xmlns="http://schemas.microsoft.com/VisualStudio/2005/CodeSnippet">
+  <CodeSnippet Format="1.0.0">
+    <Header>
+      <Title>cberr</Title>
+      <Description>Code Snippet for an error handling callback function</Description>
+      <Shortcut>cberr</Shortcut>
+      <Author>Microsoft Corporation</Author>
+      <SnippetTypes>
+        <SnippetType>Expansion</SnippetType>
+      </SnippetTypes>
+    </Header>
+    <Snippet>
+      <Declarations>
+        <Literal>
+          <ID>e</ID>
+          <Default>e</Default>
+        </Literal>
+      </Declarations>
+      <Code Language="Nodejs">
+        <![CDATA[var hadError = function ($e$, response) {
+    console.error($e$);$end$
+    response.end('Server Error');
+};
+]]>

```

Comment:
Let's remove this snippet - it covers too specific of a scenario, so I don't think it'll get much use.